cl/users/views.py

```diff
@@ -88,6 +88,12 @@ def view_favorites(request):
         elif favorite.docket_id:
             key = "Dockets"
         favorite_forms[key].append(FavoriteForm(instance=favorite))
+    docket_search_string = " or ".join(
+        [
+            "docket_id:" + str(a.instance.docket_id.pk)
+            for a in favorite_forms["Dockets"]
+        ]
+    )
```

Comment:
I think you could simplify this and the lines down below. Something like:

```suggestion
    docket_search_string = "/?type=r&q=docket_id:(%s)" % " OR ".join(favorite_forms["Dockets"]
```

I think that accomplishes the same idea?

Reply:
You need the str(a.instance.docket_id.pk) and I don't see a simple way to get it without list comprehension. I can combine the two lines, though.

Reply:
Right, OK, then the one other change I'd suggest is that the query should be `docket_id:(ID OR ID2 OR ID3)`. And what's with the trailing `and` you've got?

Reply:
The `and` is so you can add a string to the end of it to search, wasn't quite sure if that happens properly without that.

Reply:
Ah. No, it shouldn't be needed, I don't think.